handlers/main.yml

```diff
@@ -2,15 +2,10 @@
 # File: main.yml - Handlers for Consul
 
 - name: restart consul
-  service:
-    name: consul
-    state: restarted
+  include: restart_consul.yml
```

Comment:
Having OS-contextual handlers is awkward. I attempted to do this with ansible blocks but they're unsupported. This is clunkier than ideal, but is the best option I could find that works.